etna/transforms/timestamp/holiday.py

```diff
@@ -10,23 +11,45 @@
 from etna.transforms.base import IrreversibleTransform
 
 
+class HolidayTransformMode(str, Enum):
+    """Enum for different imputation strategy."""
+
+    binary = "binary"
+    category = "category"
+
+    @classmethod
+    def _missing_(cls, value):
+        raise NotImplementedError(
+            f"{value} is not a valid {cls.__name__}. Supported mode: {', '.join([repr(m.value) for m in cls])}"
+        )
+
+
 class HolidayTransform(IrreversibleTransform, FutureMixin):
-    """HolidayTransform generates series that indicates holidays in given dataframe."""
+    """
+    HolidayTransform generates series that indicates holidays in given dataframe.
+    Can either show holiday presence or their names (NO_HOLIDAY indicates absence).
+    """
+
+    NO_HOLIDAY: str = "NO_HOLIDAY"
```

Comment:
I'm not sure that it should be public and written in capitalized way. Let's name it `_no_holiday_name`.